<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransferManagerItem</class>
 <widget class="QWidget" name="TransferManagerItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>768</width>
    <height>48</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>768</width>
    <height>48</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">*
{
  font-family: &quot;Lato&quot;;
}

TransferManagerItem
{
	width: 756px; 
	heigth: 48px; 
	min-width: 756px; 
	min-heigth: 48px; 
	max-width: 756px; 
	max-heigth: 48px; 
	background-color: transparent;
}

#tFileType
{
	width: 48px; 
	heigth: 48px; 
	min-width: 48px; 
	min-heigth: 48px; 
	max-width: 48px; 
	max-heigth: 48px; 
	qProperty-iconSize: 48px;
	background: transparent;
}

QPushButton, QToolButton
{
	border: none;
}

QPushButton
{
	background: transparent;
}

#wName
{
	width: 218px; 
	heigth: 48px; 
	min-width: 218px; 
	min-heigth: 48px; 
	max-width: 218px; 
	max-heigth: 48px; 
}

#lTransferName
{
  height: 20px;
  color: #333333;
  font-size: 14px;
}

QProgressBar
{
    height: 4px;
    min-height: 4px;
    max-height: 4px;
    border:0px;
    border-radius: 2px;
    background-color: #EEEEEE;
    margin-bottom: 4px;
}

QProgressBar::chunk
{
  background-color: #6FD7FF;
  border-radius: 2px;
  border:0px;
}
#lDone, #lTotal
{
  font-size: 12px;
  font-weight: bold;
}
#lDone
{
  color: #999999;
 qProperty-alignment: 'AlignRight | AlignVCenter';
}

#lTotal
{
  color: #333333;
 qProperty-alignment: 'AlignLeft | AlignVCenter';
}

#wSize, #bSpeed
{
    width: 100px;
    min-width: 100px;
    max-width: 100px;
}

#bSpeed
{
  color: #333333;
  font-size: 12px;
  font-weight: bold;
 qProperty-alignment: 'AlignLeft | AlignVCenter';
}

#wStatus
{
    width: 120px;
    min-width: 120px;
    max-width: 120px;
}

#lStatus
{
  color: #00D26D;
  font-size: 12px;
  font-weight: bold;
}

#lQueued
{
  width: 43px;
  color: #666666;
  font-size: 12px;
  font-weight: bold;
}

#lRemainingTime
{
    width: 92px;
    min-width: 92px;
    max-width: 92px;
 color: #333333;
  font-size: 12px;
  font-weight: bold;
}

#tPauseTransfer, #tCancelTransfer
{
	width: 16px; 
	heigth: 16px; 
	min-width: 16px; 
	min-heigth: 16px; 
	max-width: 16px; 
	max-heigth: 16px; 
	background-color: #EEEEEE;
	qProperty-iconSize: 16px;
	border-radius: 4px;
	margin-right: 12px;
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="sTransferState">
     <property name="minimumSize">
      <size>
       <width>768</width>
       <height>48</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>768</width>
       <height>48</height>
      </size>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="stateActive">
      <property name="minimumSize">
       <size>
        <width>768</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>768</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QToolButton" name="tFileType"/>
       </item>
       <item>
        <widget class="QWidget" name="wName" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="lTransferName"/>
          </item>
          <item>
           <widget class="QProgressBar" name="pbTransfer">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="invertedAppearance">
             <bool>false</bool>
            </property>
            <property name="format">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wSize" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="lDone"/>
          </item>
          <item>
           <widget class="QLabel" name="lTotal"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="bSpeed"/>
       </item>
       <item>
        <widget class="QWidget" name="wStatus" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="lStatus"/>
          </item>
          <item>
           <widget class="QLabel" name="lQueued">
            <property name="text">
             <string>Queued</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lRemainingTime"/>
       </item>
       <item>
        <widget class="QToolButton" name="tPauseTransfer">
         <property name="icon">
          <iconset resource="../Resources_linux.qrc">
           <normaloff>:/images/ico_pause_transfers_state.png</normaloff>:/images/ico_pause_transfers_state.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tCancelTransfer">
         <property name="icon">
          <iconset resource="../Resources_linux.qrc">
           <normaloff>:/images/clear_item_ico.png</normaloff>:/images/clear_item_ico.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stateCompleted">
      <property name="minimumSize">
       <size>
        <width>784</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>784</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>22</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="wContainerLeftCompleted" native="true">
         <property name="minimumSize">
          <size>
           <width>550</width>
           <height>38</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>38</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widgetCompleted" native="true">
            <property name="minimumSize">
             <size>
              <width>550</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>550</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="spacing">
              <number>1</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="lTransferTypeCompleted">
               <property name="minimumSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="lFileTypeCompleted">
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>20</width>
                 <height>22</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lTransferNameCompleted">
               <property name="minimumSize">
                <size>
                 <width>375</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>375</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">padding-bottom: 1px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>30</width>
                 <height>16</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="lCompleted">
               <property name="minimumSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="iconSize">
                <size>
                 <width>12</width>
                 <height>12</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lTotalCompleted">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>80</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">padding-left: 8px;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="alignment">
                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="wContainerRightCompleted" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>13</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="lActionTypeCompleted">
            <property name="minimumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lRemainingTimeCompleted">
            <property name="minimumSize">
             <size>
              <width>90</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>90</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="lCancelTransferCompleted">
            <property name="minimumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../Resources_linux.qrc">
              <normaloff>:/images/clear_item_ico.png</normaloff>:/images/clear_item_ico.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>12</width>
              <height>12</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources_linux.qrc"/>
 </resources>
 <connections/>
</ui>
